app/styles/component/_affiliate-unit.scss

```diff
@@ -60,12 +54,11 @@
 }
 
 .article-content .affiliate-unit {
-  background-image: linear-gradient(to bottom, $wds-color-white, $wds-color-off-white);
   margin: 0 -.75rem 48px;
 
   &__slant {
-    background-image: linear-gradient(-5deg, $wds-color-white 0%, $wds-color-white 50%, $wds-color-off-white 50%);
-    height: 36px;
+    background-image: linear-gradient(-5deg, rgba(255, 255, 255, 0) 5%, $wds-color-white 60%, rgba(255, 255, 255, 0) 60%);
```

Comment:
Are you sure this isn't supposed to use the off-white color at all?

Reply:
Zeplin mocks don't reference off-white.
https://zpl.io/bLGEQPY
zpl://screen?pid=5c34f8570b5f05bdc9806cb9&sid=5db233296a01342c331fe781